### DISCUSSION TITLE
Используете ли вы функцию "Связать громкость перевода и видео"?

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#265] [Polls]
Хотелось бы узнать, используете ли вы функцию "Связать громкость перевода и видео" или нет, а так же узнать ваше мнение на счет того нужна ли она в расширение.

Для тех, кто не знает, что она делает: "Связать громкость перевода и видео" - функция, которая позволяет связать ползунок громкости перевода и ползунок громкости оригинального видео, тем самым двигая 1 ползунок будет увеличиваться/уменьшаться громкость обоих ползунков (ниже есть демо).

![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/d84d150d-def7-4baa-a477-215a8383d181)

https://github.com/ilyhalight/voice-over-translation/assets/62353659/caf3e464-d6b5-47f5-8018-7dbeddce94cb

**Comment by FiLinX** [comment]
Вроде бы полезная функция, и я помню я её даже просил) Но вот когда она ещё только появилась и я её пробовал, видимо что-то у меня неправильно срабатывало и всё сбивалось, то есть после попытки покрутить колесом мыши, назад всё не возвращалось - громкость оригинала перекрывала громкость перевода, и поэтому я больше так эту функцию и не пробовал держа её всегда отключенной)
Но всё же инструмент такой или подобный нужен наверно..
Хотя мне сейчас чаще нужен как бы временный инвертор - чтоб по удержанию какой либо клавиши уровень оригинала повышался а перевод затихал.. Просто в последнее время часто попадаются иноземные ролики, где в сюжете имеются вставки интервью из русскоязычных видео (они там обычно субтитрами на инглише, а яндекс их не передублирует), и вот их звук в эти моменты остаётся очень тихим так как я использую установку 5% уровня оригинала..
Вот и подумалось об какой либо кнопке, по удержанию которой перевод бы типа временно "отключался" и звук в эти моменты шёл как с оригинала)
